test/autodiff/ad.jl

```diff
@@ -593,7 +597,7 @@ for V in spacelist
                     # TODO: I'm not sure how to properly test with spaces that might change
                     # with the finite-difference methods, as then the jacobian is ill-defined.
 
-                    trunc = truncrank(round(Int, min(dim(domain(t)), dim(codomain(t))) ÷ 2))
+                    trunc = truncrank(max(2, round(Int, min(dim(domain(t)), dim(codomain(t))) * (3 / 4))))
```

Comment:
This is interesting: given that our `dim` can return floating point numbers for the anyonic case, do we also need to allow floating point inputs for `truncrank`?